blocks/amp-story/amp-story-cta-layer.js

```diff
@@ -9,7 +9,8 @@ const {
 } = wp.blocks;
 const {
 	InspectorControls,
-	InnerBlocks
+	InnerBlocks,
+	RichText
```

Comment:
AC3 of the issue was to also pre-populate CTA block with a `core/button` block -- is this something that doesn't seem to be possible at this moment? I thought I saw a template variable somewhere yesterday but not seeing it today anymore.